test/bloom-test.js

```diff
@@ -180,4 +180,29 @@ describe('Bloom', function() {
 
     assert(!filter.test('foobar 49', 'ascii'));
   });
+
+  // Regression test for missing filter.size setting
+  // in Bloom.prototype.fromReader
+  it('should serialize and inject properties from ' +
+          'serialized data correctly', () => {
+    // Hardcoded sample data for filter
+    var size = 32;
+    var filter = Buffer.from('deadbeef', 'hex');
+    var n = 9;
+    var tweak = 172012938;
+    var update = Bloom.flags.NONE;
+
+    var fooBloom = new Bloom(size, n, tweak, update);
+    fooBloom.filter = filter;
+    var serialized = fooBloom.toRaw();
+    var barBloom = Bloom.fromRaw(serialized);
```

Comment:
Consider using `const` for unchanging variables and `let` for changing variables.